tutorials/tools/Text_Normalization_Tutorial.ipynb

```diff
@@ -0,0 +1,472 @@
+      "pygments_lexer": "ipython3",
+      "version": "3.7.9"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "x0DJqotopcyb"
+      },
+      "source": [
+        "\"\"\"\n",
+        "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+        "\n",
+        "Instructions for setting up Colab are as follows:\n",
+        "1. Open a new Python 3 notebook.\n",
+        "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+        "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+        "4. Run this cell to set up dependencies.\n",
+        "\"\"\"\n",
+        "# If you're using Google Colab and not running locally, run this cell\n",
+        "\n",
+        "# install NeMo\n",
+        "BRANCH = 'main'\n",
+        "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[nlp]"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "CH7yR7cSwPKr"
+      },
+      "source": [
+        "import json\n",
+        "import os\n",
+        "import wget\n",
+        "import numpy as np\n",
+        "import inspect\n",
+        "import regex as re\n"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xXRARM8XtK_g"
+      },
+      "source": [
+        "# Introduction\n",
+        "Text normalization for TTS converts text into its verbalized form. That is, tokens belonging to special semiotic classes, e.g. numbers/abbreviations, will be converted into their spoken form. For example, \"10:00\" -> \"ten o'clock\", \"10:00 a.m.\" -> \"ten a m\", \"10kg\" -> \"ten kilograms\".\n",
+        "\n",
+        "This tutorial shows how to use the NeMo rule-based text normalization system.\n",
+        "Similar to [The Google Kestrel TTS text normalization\n",
+        "system](https://www.researchgate.net/profile/Richard_Sproat/publication/277932107_The_Kestrel_TTS_text_normalization_system/links/57308b1108aeaae23f5cc8c4/The-Kestrel-TTS-text-normalization-system.pdf), the NeMo rule-based system is devided into a tagger and a verbalizer: the tagger is responsible for detecting and classifying semiotic classes in the underlying text, the verbalizer takes the output of the tagger and carries out the normalization. The system is designed to be easily debuggable and extendable by more rules.\n",
+        "We provided the a set of rules that covers the majority of cases as found in the [Google Text normalization dataset](https://www.kaggle.com/richardwilliamsproat/text-normalization-for-english-russian-and-polish) for the English language. As with every language there is a long tail of special cases.\n",
```

Comment:
Could you add more details on the 2 step processes: tagger and then normalization maybe add an example or reference the Evaluation data section for more details?

Reply:
done